org.envirocar.app/res/layout/activity_signup.xml

```diff
@@ -117,7 +117,8 @@
             app:layout_constraintStart_toStartOf="@id/activity_signup_guideline_left"
             app:layout_constraintTop_toBottomOf="@+id/activity_signup_email_input_layout"
             app:layout_constraintWidth_max="500dp"
-            app:passwordToggleEnabled="true">
+            app:passwordToggleEnabled="true"
+            app:passwordToggleTint="#000">
```

Comment:
The Eye Icon throughout the Input Fields over the android application have been used Grey color instead of Black. I think it would be inconsistent to use only Black color on certain Eye Icons. Can you please provide relevant Screenshots of the Changes introduced, and if Interface is consistent and good enough with the darker Eye Icons, we can proceed.